cypress/integration/data-import/upload-file-update.spec.js

```diff
@@ -0,0 +1,311 @@
+      settingsDataImport.goToActionProfile();
+      ActionProfiles.createActionProfile(profile.actionProfile, profile.mappingProfile);
+      ActionProfiles.checkActionProfilePresented(profile.actionProfile.name);
+    });
+
+    // create Match profile
+    const collectionOfMatchProfiles = [
+      {
+        matchProfile: { profileName: matchProfileNameForInstance,
+          incomingRecordFields: {
+            field: '001'
+          },
+          existingRecordFields: {
+            field: '001'
+          },
+          matchCriterion: 'Exactly matches',
+          existingRecordType: 'MARC_BIBLIOGRAPHIC' }
+      },
+      {
+        matchProfile: { profileName: matchProfileNameForHoldings,
+          incomingRecordFields: {
+            field: '901',
+            subfield: 'a'
+          },
+          matchCriterion: 'Exactly matches',
+          existingRecordType: 'HOLDINGS' }
+      },
+      {
+        matchProfile: {
+          profileName: matchProfileNameForItem,
+          incomingRecordFields: {
+            field: '902',
+            subfield: 'a'
+          },
+          matchCriterion: 'Exactly matches',
+          existingRecordType: 'ITEM'
+        }
+      }
+    ];
+
+    settingsDataImport.goToMatchProfile();
+    collectionOfMatchProfiles.forEach(profile => {
+      MatchProfiles.createMatchProfile(profile.matchProfile);
+    });
+
+    // create Job profile
+    const jobProfileForUpload = {
+      ...NewJobProfile.defaultJobProfile,
+      profileName: jobProfileName
+    };
+    settingsDataImport.goToJobProfile();
+    jobProfiles.openNewJobProfileForm();
+    NewJobProfile.fillJobProfile(jobProfileForUpload);
+    NewJobProfile.linkMatchAndActionProfiles(matchProfileNameForInstance, actionProfileNameForInstance);
+    NewJobProfile.linkMatchAndActionProfiles(matchProfileNameForHoldings, actionProfileNameForHoldings);
+    NewJobProfile.linkMatchAndActionProfiles(matchProfileNameForItem, actionProfileNameForItem);
+    NewJobProfile.clickSaveAndCloseButton();
+    jobProfiles.waitLoadingList();
+    jobProfiles.checkJobProfilePresented(jobProfileName);
+
+    // upload the exported marc file
+    dataImport.goToDataImport();
+    dataImport.uploadFile(nameForExportedMarcFile);
+    jobProfiles.searchJobProfileForImport(jobProfileName);
+    jobProfiles.runImportFile(nameForExportedMarcFile);
+    logs.openJobProfile(nameForExportedMarcFile);
+    logs.checkIsInstanceUpdated();
+
+    // delete generated profiles
+    jobProfiles.deleteJobProfile(jobProfileForUpload.profileName);
+    collectionOfMatchProfiles.forEach(profile => {
+      MatchProfiles.deleteMatchProfile(profile.matchProfile.profileName);
+    });
+    collectionOfProfiles.forEach(profile => {
+      FieldMappingProfiles.deleteFieldMappingProfile(profile.mappingProfile.profileName);
+      ActionProfiles.deleteActionProfile(profile.actionProfile.profileName);
+    });
```

Comment:
```suggestion
  // delete downloads folder and created files in fixtures
            FileManager.deleteFolder(Cypress.config('downloadsFolder'));
            FileManager.deleteFile(`cypress/fixtures/${nameForExportedMarcFile}`);
            FileManager.deleteFile(`cypress/fixtures/${nameForCSVFile}`);
    });
```

Reply:
thanks, I added this part of code in my test